src/legacy/core_plugins/metrics/public/components/lib/get_interval.js

```diff
@@ -19,8 +19,8 @@
 import moment from 'moment';
 import { i18n } from '@kbn/i18n';
 import { pluck, get, clone } from 'lodash';
-import { relativeOptions } from '../../../../../ui/public/timepicker/relative_options';
 
+import { relativeOptions } from './relative_options';
```

Comment:
Not sure that we need the '**relativeOptions**' in TSVB at all. 

I added a breakpoint at line # 40 and found that the variable 'unit' has the following value:

![image](https://user-images.githubusercontent.com/20072247/61636150-b3ece300-ac9d-11e9-8644-ae8665d7a6ce.png)

What if you do something like: 
![image](https://user-images.githubusercontent.com/20072247/61636578-91a79500-ac9e-11e9-9d01-eff5c6b77dae.png)


@markov00 @timroes  any ideas?

Reply:
I don't mind removing it.
But would like your suggestion, as I'm unfamiliar with this code.

Reply:
You were absolutely right.
I removed relative options and replaced with a use of `Object.keys` similar to what you suggested @alexwizp